server/plugins/renderer/renderer.module.js

```diff
@@ -198,7 +201,53 @@ internals.parseResponse = async (bcAppData, request, response, responseArgs) =>
         cache.put(dataRequestSignature, { response2 }, internals.cacheTTL);
     }
 
-    return internals.getPencilResponse(response2.data, request, response2, configuration);
+    const templateFile = response2.data.template_file;
+    const entityId = response2.data.entity_id;
+
+    const pageType = getPageType(templateFile);
+    let regionResponse = [];
+
+    if (pageType !== null) {
+        // create request signature and use cache, if available
+        const graphQLUrlSignature = internals.sha1sum(internals.options.storeUrl + '/graphql');
+        const graphQLQuerySignature = internals.sha1sum(pageType + entityId);
+        const graphQLDataReqSignature = `graphql:${graphQLUrlSignature}${graphQLQuerySignature}`;
+        const cachedGraphQLResponse = cache.get(graphQLDataReqSignature);
+
+        if (internals.options.useCache && cachedGraphQLResponse) {
+            ({ regionResponse } = cachedGraphQLResponse);
```

Comment:
What does this mean?

Reply:
It's extracting cached value of `regionResponse` instead of making a graphQL call.